src/utils/fileUtils.js

```diff
@@ -119,4 +123,18 @@ const genFileInfo = function(obj) {
 	return fileInfo
 }
 
-export { encodeFilePath, extractFilePaths, sortCompare, genFileInfo }
+/**
+ * Generate absolute dav remote path of the file
+ * @param {object} fileInfo The fileInfo
+ * @returns {string}
+ */
+function genDavPath(fileInfo) {
```

Comment:
More suitable: getDavPath

```suggestion
function getDavPath(fileInfo) {
```

Reply:
Also, please keep the same function declaration format

```suggestion
const getDavPath = function(fileInfo) {
```